api/v1/installation_types.go

```diff
@@ -178,6 +179,35 @@ type InstallationSpec struct {
 	// +kubebuilder:validation:Enum=Enabled;Disabled
 	// +optional
 	FIPSMode *FIPSMode `json:"fipsMode,omitempty"`
+
+	// Logging Configuration for Components
+	// +optional
+	Logging *Logging `json:"logging,omitempty"`
+}
+
+type Logging struct {
+	// Customized logging specification for calico-cni plugin
+	// +optional
+	CNILogging *CNILogging `json:"cniLogging,omitempty"`
```

Comment:
I would personally call this field `CNI` - it's already within the `Logging` struct so we don't need to say logging again.

```yaml
logging:
  cni:
    logSeverity: Info
```

You can (and should) still call the struct `CNILogging`, I'm just suggesting renaming the field name (and json tag)

Reply:
Done.